app/views/provider/admin/dashboards/show.html.slim

```diff
@@ -4,6 +4,11 @@
 - content_for :javascripts do
   = javascript_packs_with_chunks_tag 'dashboard'
 
+button class="pf-c-button pf-m-primary pf-c-button__float-button" type="button"
+  span class="pf-c-button__icon pf-m-start"
+    i class="fas fa-info-circle" aria-hidden="true"
+  a href="/p/admin/quickstarts" Quickstarts
```

Comment:
I think this is too much and the flash notice should be enough, but if that's what they want...